tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "compilerOptions": {
+      /* Basic Options */
+      "target": "ES6",
+      "module": "system",
+      "lib": ["es2017", "dom"],
+      "allowJs": true,
+      "checkJs": false,
```

Comment:
Shouldn't be true?

Reply:
the doc generation doesn't work if this option sis set to true (too many errors)